src/scss/components/_ratings.scss

```diff
@@ -0,0 +1,65 @@
+$star-count: 6;
+$star-size-small: 14px;
+$star-size-medium: 28px;
+$star-size-large: 42px;
+
+.c-ratings {
```

Comment:
Think lost my comment from before - would be good to keep this singular as it's only one rating.

So `c-rating` rather than `c-ratings`.  Otherwise looks good and happy to approve